adhocracy4/actions/verbs.py

```diff
@@ -0,0 +1,12 @@
+# http://activitystrea.ms/registry/verbs/
+
+CREATE = 'created'
```

Comment:
Left hand values are in pastense to keep it compatible to an older version of OPIN actions.